### PR TITLE
Change permission for /etc/rsyslog.d

### PR DESCRIPTION
**Pull request by mgirgisf**

### CODE REVIEW

**Comment by mgirgisf** [comment]
recheck

**Comment by jlarriba** [comment]
/lgtm

We need this as we are getting blocked from permissions on that dir on downstream testing.

File: roles/edpm_telemetry_logging/tasks/configure.yml [hunk 1]
```diff
@@ -22,27 +22,31 @@
...
 
     - name: Copy Openshift CA to the node
-      become: true
       ansible.builtin.copy:
         src: "{{ edpm_telemetry_logging_config_src }}/ca-openshift.crt"
         dest: "/etc/pki/rsyslog/ca-openshift.crt"
         mode: "0644"
         remote_src: "{{ telemetry_test | default('false') }}"
 
+    - name: Change permissions of /etc/rsyslog.d to 755
```
Comment (elfiesmelfie) [inline roles/edpm_telemetry_logging/tasks/configure.yml:43]:
```suggestion
    - name: Ensure that /etc/rsyslog.d/ exists with the right permissions
```

**Review by elfiesmelfie — CHANGES_REQUESTED** [review]

**Review by elfiesmelfie — APPROVED** [review]

**Review by jpodivin — APPROVED** [review]

**Pull request merged**